Templates/ScriptCanvasNode/Template/README.md

```diff
@@ -0,0 +1,36 @@
+## How to use this template
+
+Use this template to produce a Script Canvas project that may hold Script Canvas nodes. 
```

Comment:
Technically this isn't a project, this would be adding a custom Script Canvas node to a custom gem.

Reply:
It adds a project in the Visual Studio sense (aka a vcproj), I'm open to rephrase it.

Reply:
Maybe something like "a build target that may hold a library of Script Canvas nodes"?